src/driver/sqlserver/SqlServerQueryRunner.ts

```diff
@@ -269,6 +269,22 @@ export class SqlServerQueryRunner
 
             if (raw?.hasOwnProperty("recordset")) {
                 result.records = raw.recordset
+                // mssql driver will return an array if duplicate columns exist
+                // The following will search for properties that return as an array, and then update the property with the last value
```

Comment:
why exactly the _last value_? Why not the first, second or n-th?

Reply:
@pleerock I think it's for consistency with other database drivers.

Reply:
thanks @Ginden . What's your opinion on this PR?

Reply:
I have mixed feelings. Consistency is good, but underlying issue of duplicated column names looks iffy.